packages/frontend/src/features/notification/stores/useNotificationStore.tsx

```diff
@@ -0,0 +1,69 @@
+import { create } from 'zustand'
+import { NotificationAction } from '../types/NotificationTypes'
+
+interface Notification {
+    id: number
+    type: string
+    message: string
+    time: string
+    isRead: boolean
+    actions?: NotificationAction[]
```

Comment:
Maybe don't need `actions` in the `Notification` interface because you access `actions` from `NofiticationConfig` interface in the `NotificationItem` component.

Reply:
removed.